# Configuration for `pub_rgbd_and_cloud.py`.

ros_topic_namespace: "test_data"
publish_rate: 0.5 # How many data to publish per second.
is_loop_forever: True # After publishing all images, loop back to the 1st one.
frame_id: "head_camera" # Name of the camera frame.

color:
  is_publish: True
  folder: "data/color/"
  ros_topic: "color"

depth:
  is_publish: True
  folder: "data/depth/"
  depth_unit: 0.001 # If depth[i,j]=D, then this pixel's depth is (depth_unit*D) meters.
  depth_trunc: 3.0 # Depth value larger than ${depth_trunc} meters gets truncated to zero.
  ros_topic: "depth"

camera_info:
  is_publish: False
  file: ""
  ros_topic: "camera_info"

point_cloud:
  is_publish: False
  ros_topic: "point_cloud"
